units/host_monitoring_idle_termination.go

```diff
@@ -194,12 +194,12 @@ func (j *idleHostJob) checkAndTerminateHost(ctx context.Context, h *host.Host) e
 
 //checkTerminationExemptions checks if some conditions apply where we shouldn't terminate an idle host,
 //and returns true if some exemption applies
-func (j *idleHostJob) checkTerminationExemptions(ctx context.Context, h *host.Host) (bool, error) {
+func checkTerminationExemptions(ctx context.Context, h *host.Host, env evergreen.Environment, jobName string, jid string) (bool, error) {
```

Comment:
nit: `jobName` should be `jobType` because amboy already refers to the logical job ID the "job name".